nbs/mint.json

```
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://mintlify.com/schema.json",
+  "name": "Nixtla",
+  "logo": {
+    "light": "/light.png",
+    "dark": "/dark.png"
+  },
+  "favicon": "/favicon.svg",
+  "colors": {
+    "primary": "#0E0E0E",
+    "light": "#FAFAFA",
+    "dark": "#0E0E0E",
+    "anchors": {
+      "from": "#2AD0CA",
+      "to": "#0E00F8"
+    }
+  },
+  "topbarCtaButton": {
+    "type": "github",
+    "url": "https://github.com/Nixtla/nixtla"
```

Also not sure if this should point to each repo. stats points to nixtla/nixtla as well
```suggestion
    "url": "https://github.com/Nixtla/mlforecast"
```

---

> Also not sure if this should point to each repo

The only value that matters in the `mint.json` is actually only the `navigation` field. All the other fields are there just so that the preview that i mentioned here will work:

> Run bash ./preview-mintlify-docs.bash to get a preview of the docs locally. Note - the preview will only show mlforecast but the Nixtla deployment will be a frankenstein merged version of all 4 docs instances. I created this command so that developers can preview & sanity check their changes before pushing.

---

i updated the values nonetheless!